bulktest/batch_lib.inc

```diff
@@ -964,7 +964,10 @@ function prettyType($mimeType, $ext) {
 	else if ( "xml" === $ext ) {
 		return "xml";
 	}
-	else if ( false !== strpos($mimeType, "flash") || "mp4" === $ext || "swf" === $ext || "f4v" === $ext || "flv" === $ext ) {
+    //Video extensions mp4, webm, ts, m4v, m4s, m4v, mov, ogv
+	else if ( false !== strpos($mimeType, "flash") || (true === strpos($mimeType, "webm") || (true === strpos($mimeType, "mp4") || (true === strpos($mimeType, "flv"))
+         "mp4" === $ext || "webm" === $ext || "ts" == $ext || "m4v" === $ext || "m4s" === $ext || "mov" === $ext || "ogv" === $ext ||
```

Comment:
Never seen `ts` before. Turns out it's [used by DVDs](https://fileinfo.com/extension/ts). Is this really something we need to check? Possible collisions with TypeScript files?

Reply:
ts is the third most common extension according to the current data. It's used in DVB streams and in VOD platforms.

From the query I ran:

Extention | MIMEType | Count 
-----------------------------
mp4 | video/mp4 | 142048
  | video/webm | 11836  
ts | video/MP2T | 11826